lib/Entry/Json.hs

```diff
@@ -11,20 +11,21 @@ import Data.Aeson qualified as A
 import Data.Aeson.Lens qualified as A
 import Data.Either.Extra (eitherToMaybe)
 import Data.HashMap.Strict qualified as HS
+import Data.Text (Text)
 import Data.Text qualified as T
 import Data.Time.Format.ISO8601 (iso8601ParseM)
-import Entry (Entry, FieldValue(FieldValue))
+import Entry (Entry, EntryConvertible, Field, FieldValue(FieldValue))
 import Entry qualified as E
 import Fmt (pretty)
 
 newtype JsonEntry = JsonEntry A.Value
   deriving stock (Show)
   deriving newtype (A.ToJSON, A.FromJSON)
 
-fieldConverter :: (Prism A.Value A.Value E.Field E.Field)
+fieldConverter :: (Prism A.Value A.Value Field Field)
 fieldConverter = prism' to from
   where
-    to :: E.Field -> A.Value
+    to :: Field -> A.Value
     to field =
       A.object
       [ "date_modified" A..= (field ^. E.dateModified)
```

Comment:
Maybe we can do the same thing with `A.Value` and `A..=` (for e.g. hide `.=` from `Control.Lens`)

Reply:
Done, did the same with `Toml..=`